guide/interactions/registering-slash-commands.md

```diff
@@ -0,0 +1,124 @@
+# Registering slash commands
+
+Discord provides users the option to create client-integrated slash commands. In this section you will be learning how to register these commands using discord.js!
+
+::: tip
+If you already have slash commands set-up for your application and want to learn to respond to them, refer to [the following page](/interactions/replying-to-slash-commands/).
+:::
+
+
+## Global commands
+
+First up, we'll introduce you to global application commands, these type of commands will be available on every guild your application is authorized on with the `applications.commands` oauth scope and are also available in its DMs.
+
+::: warning
+Global commands are cached for one hour. That means that new global commands will fan out slowly across all guilds, and will be guaranteed to be updated in an hour. 
+:::
+
+::: tip
+Guild commands update instantly. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.
+:::
```

Comment:
This could be combined into one block

```suggestion
::: tip
Global commands are cached for one hour. That means that new global commands will fan out slowly across all guilds and will be guaranteed to be updated in an hour. Guild commands however update instantly, so we recommend you use these for quick testing and global commands when they're ready for public use.
:::
```